src/GogeDao.sol

```diff
@@ -1,1087 +1,793 @@
+/// @title Doge Day Son Dao Contract.
+/// @notice This contract is a governance contract which acts as a layer that sits on top of a governance token. This contract allows holders of the governance token
+///         to create proposals of a chosen pollType. Each pollType has unique parameters which, if passes quorum, will result in a function call to the governance token.
+/// @author Chase Brown
+contract GogeDAO is Owned {
 
-All current issues fixed in contract so we pass an audit with 100% :D (also show we show up on token scanners without any warnings) (I believe these were the 90% scanners / ability to make taxes a very high amount, perhaps we should have a way we can never change the taxes to something over our starting tax, or the rainbow hours max %) (also the anti bot I think had something that could be changed/ the blacklisting issue) (I can’t think of anything else as of right now) 
+    // ---------------
+    // State Variables
+    // ---------------
 
-* if possible * A charity wallet that the ĐAO could control if possible (for direct action charity and also for charity donations)
-*/
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.6;
+    /// @notice Contract address of governance token.
+    address public governanceTokenAddr;
+    
+    /// @notice Unique identifier of each poll that is created.
+    uint256 public pollNum;
+    /// @notice The minimum time needed for a new poll -> default is 1 day.
+    uint256 public minPeriod = 1 days;
+    /// @notice The maximum time needed for a new poll -> default is 60 days.
+    uint256 public maxPeriod = 60 days;
+    /// @notice The minimum balance of governance token the author of a poll must be holding at the time of creation.
+    uint256 public minAuthorBal = 10_000_000 ether;
+    /// @notice The maximum amount of polls an author can have active at any given time.
+    uint256 public maxPollsPerAuthor = 1;
+    /// @notice The threshold that must be met to pass a poll
+    uint256 public quorum = 50;
 
-import "./libraries/Libraries.sol";
-import "./extensions/Ownable.sol";
-import "./extensions/IGogeERC20.sol";
+    /// @notice Amount of BNB held for marketing purposes.
+    uint256 public marketingBalance;
+    /// @notice Amount of BNB held for team pay
+    uint256 public teamBalance;
 
+    /// @notice Bool if gate keeping is enabled.
+    bool public gatekeeping = true;
+    /// @notice Bool if createPoll is callable.
+    bool public createPollEnabled;
 
-contract GogeDAO is Ownable {
-    using SafeMath for uint256;
+    /// @notice Array of team member addresses.
+    address [] public teamMembers;
+    /// @notice Array of active polls -> array of poll nums.
+    uint256 [] public activePolls;
     
-    address public governanceTokenAddr;
-    uint256 public pollNum;
-    uint256 public minPeriod = 86400;
+    /// @notice Double Mapping of pollNum to amount of votes per voter.
     mapping(uint256 => mapping(address => uint256)) public polls;
```

Comment:
Personally I think polls makes sense. It is the place where all votes per voter for each proposal can be found. Like polls on voting day. The place where votes are contained.

Reply:
Looking at definitions for proposal, poll, and vote (nouns):
- **Proposal:** a plan or suggestion, especially a formal or written one, put forward for consideration or discussion by others.
- **Poll:** the process of voting in an election
- **Vote:** a formal indication of a choice between two or more candidates or courses of action, expressed typically through a ballot or a show of hands or by voice.

A poll is the process of voting and although polls can involve votes calling the mapping `polls` doesn't make sense for the data contained or returned since it operates on an individual basis. This mapping contains the number of votes cast by a specific voter for a specific proposal. It would almost make more sense to call this mapping `votesBy` lol

Reply:
Well votesBy wouldn't make sense because it's a double mapping. First point is pollNum, then address to get amount of votes per address per poll. The 'polls' refer to the tracking of votes for all proposals. I think this is a pretty trivial change tbh